zzz_generated/controller-clusters/zap-generated/CHIPClusters.h

```diff
@@ -42,6 +46,16 @@ class DLL_EXPORT AccountLoginCluster : public ClusterBase
     CHIP_ERROR Login(Callback::Cancelable * onSuccessCallback, Callback::Cancelable * onFailureCallback,
                      chip::ByteSpan tempAccountIdentifier, chip::ByteSpan setupPIN);
 
+    // Cluster Commands (Cluster Object Implementation)
+    using OnCommandErrorCallbackFunct = std::function<void(Protocols::InteractionModel::Status, CHIP_ERROR)>;
+    using OnGetSetupPINCommandResponseCallbackFunct =
+        std::function<void(const app::clusters::AccountLogin::GetSetupPINResponseCommandParams::Type &)>;
```

Comment:
These do not need to be std::functions. The existing callbacks are fairly vanilla function callbacks, so we should preserve that.